components/brave_savings/browser/buildflags/buildflags.gni

```diff
@@ -0,0 +1,5 @@
+import("//build/config/features.gni")
+
+declare_args() {
+  brave_perf_predictor_enabled = true
```

Comment:
we have some legacy flags like this, but the convention is `enable_brave_perf_predictor`